src/main/java/com/alkemy/ong/service/impl/CategoryServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.alkemy.ong.service.impl;
+
+import com.alkemy.ong.entity.CategoryEntity;
+import com.alkemy.ong.repository.CategoryEntityRepository;
+import com.alkemy.ong.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class CategoryServiceImpl implements CategoryService {
+    @Autowired
+    CategoryEntityRepository categoryRepo;
```

Comment:
Maybe just CategoryRepository is stead of Category-Entity-repository

Reply:
it is not with that name in source

Reply:
please, continue with the naming of names that we have been working on, for example: class name + container package

Reply:
hecho en la resolucion de conflicto